ui/src/common/components/NewFeatureTag/NewFeatureTag.scss

```diff
@@ -9,3 +9,7 @@
     opacity: 1;
   }
 }
+
+.NoMargin {
```

Comment:
so this class should in the component where you're setting not here.

also `without-margin-left` would be a better name.